src/elastic-search/Elasticsearch_guide.md

```diff
@@ -0,0 +1,211 @@
+# Getting started
+
+To be able to start application with Elasticsearch you will need to:
+
+1. Start with `npm prepare:local` to executes docker-compose file which pull and run Elasticsearch cluster as a docker container.
```

Comment:
This should have an indication that it is only for dev setups, refer to the  es search docs for production setups.